Topic 3/src/main/java/bootcamp/exercises/Roman.java

```diff
@@ -0,0 +1,55 @@
+package bootcamp.exercises;
+
+public class Roman {
+	private final char[] romanNumbers= {' ','I','V','X','L','C','D','M'};
+	private final int[] numbers = {0,1,5,10,50,100,500,1000};
```

Comment:
[High] These two properties should be static also (you don't want to change them)